app/main.py

```diff
@@ -2,0 +1,25 @@
+import os
+
+
+def move_file(command: str) -> None:
+    parts = command.split()
+
+    if len(parts) != 3 or parts[0] != "mv":
+        raise ValueError(
+            "Invalid command format. Use: mv <source> <destination>"
+        )
+
+    source, destination = parts[1], parts[2]
+
+    if not os.path.isfile(source):
```

Comment:
The check `os.path.isfile(source)` ensures the source is a file. This is correct, but if directories are also to be moved, consider using `os.path.exists(source)` instead.